frontend/public/kubevirt/components/modals/delete-device-modal.jsx

```diff
@@ -64,6 +64,11 @@ class DeleteDeviceModal extends PromiseComponent {
       });
     }
 
+    const bootOrderIndex = _.get(device, 'bootOrder', -1);
+    if (bootOrderIndex !== -1) {
+      patch = patch.concat(getDeviceBootOrderPatch(vm, bootOrderIndex));
```

Comment:
just a suggestion: we could send the device path in patch format or send a type (nic, disk) and index to identify the device. It is weird to me to identify devices by bootOrder. Probably nothing wrong with that though.

Reply:
+1 for type and index (or device name)

Reply:
Done!